contents/teams/marketing/objectives.mdx

```diff
@@ -1,24 +1,137 @@
-Increase the number of people signing up for PostHog's products beyond Product Analytics.
-
-**Rationale**
-In Q4 2023, we tried a few new things and started measuring impact more seriously. We have a lot more confidence in what works well (and have covered a lot of bases with existing products), so we should use this to make products beyond analytics successful.
-
-**Projects**
-- Complete our ‘Alternatives to X’ SEO content coverage for feature flags, experimentation, and session replay (Andy)
-- Make sure we have good ‘getting started’ content (Joe)
-- Write tutorials for all our key frameworks and products (Lior)
-- Increase newsletter subs from 6k to 10k (Andy)
-- Repeat our success with Theo on TikTok with new influencers (Ian)
-- Rework all our paid ads copy now we’ve cut back to just Google Search (Charles)
-- Get 50% of the W24 batch to sign up (Joe)
-- Make sure we keep measuring impact regularly for all of our content (Everyone)
-- Book all the billboards and/or other out-of-home spots we want for a January 2025 campaign (Lottie)
-
-**Anti-goals**
-- Building a community (Website & Docs are working on this)
-- Getting good at social media (maintaining steady state for now)
-- Leading events (but we'll help CS a bit)
-
-**Output metrics we care about**
+Increase awareness of PostHog within our ICP and ensure PostHog is a brand that sparks conversation and joy.
+
+## Projects
+
+- [Docs that spark joy](#docs-that-spark-joy-andy) (Andy)
+- [Drive word of mouth through improving support](#drive-word-of-mouth-through-improving-support-joe) (Joe)
+- [Supporting CS in their sales-led activities](#supporting-cs-in-their-sales-led-activities-joe) (Joe)
+- [Become default analytics tool for new AI startups](#become-default-analytics-tool-for-new-ai-startups-lior) (Lior)
+- [Finish alternatives guides](#finish-alternatives-guides-andy) (Andy)
+- [Scale newsletter](#scale-newsletter-james) (James)
+- [Level up branding](#level-up-branding-lottie) (Lottie)
+- [Ship awesome technical blogs](#ship-awesome-technical-blogs-ian) (Ian)
+- [Hiring plan for marketing team](#hiring-plan-for-marketing-team-charles) (Charles)
+
+### Docs that spark joy (Andy)
+
+**Rationale:** Some of our documentation has fallen behind the product and there are pockets that fall well short of our current standards. This is a reputational risk if left unresolved.
+
+**Contributors:** Andy, Ian, Lior + product teams (hopefully)
+
+**What we'll ship: (WIP)**
+- Best practice and use case guides for:
+  - Product analytics
+  - Session replay
+  - Feature flags
+  - A/B testing
+  - Data warehouse (if/when ready)
+- Much better HogQL docs
+- Web analytics docs (if/when ready)
+- Update or remove any old/low quality docs
+- Optimize docs index pages for desire paths / FAQs
+
+**We'll know we're successful when:** Feedback from community / on social that our docs are great, and feedback from product teams.
```

Comment:
Could we use the docs helpful survey to give this a hard metric? Feels like we should, since we have that function, we have existing data to judge against, and we can use that info to target further changes.

I know we don't _need_ a hard metric for things always, but this is somewhere we can definitely have one and use it to ensure we're making an impact.